pkg/sdk/custom_shapes.go

```diff
@@ -18,8 +18,10 @@ import (
 	"fmt"
 
 	awssdkmodel "github.com/aws-controllers-k8s/code-generator/pkg/api"
+	simpleschema "github.com/kro-run/kro/pkg/simpleschema"
 
 	ackmodel "github.com/aws-controllers-k8s/code-generator/pkg/model"
+	apiextv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
```

Comment:
The standard is first standard libraries, then external, then internal

```suggestion
	simpleschema "github.com/kro-run/kro/pkg/simpleschema"
	apiextv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"

	ackmodel "github.com/aws-controllers-k8s/code-generator/pkg/model"
	awssdkmodel "github.com/aws-controllers-k8s/code-generator/pkg/api"
```